### DISCUSSION TITLE
Is it possible to remove translucency from the background of the translated text?

### DISCUSSION THREAD

**Discussion by armageddon735** [killkimno/MORT#59] [Q&A] [answered]
I'm terribly sorry for my stupidity, but I can't find the setting responsible for this, and I'm already starting to think if there is one at all, I use the translator in overlay mode, and the text is displayed on top of the original for convenience, but due to the fact that the background is not solid but transparent, it's not very convenient to read

**Comment by kmonkeyhead** [comment, marked as answer]
yes you can change background color and opacity
1. MORT -> Text Tab
2. Click Color -> Background rectangle
3. now you can change color editor window
4. Click "투명도" then you can change alpha value
![image](https://github.com/user-attachments/assets/9de335c0-2e69-4f01-908a-74521855a29a)

**Reply by armageddon735** [reply]
thank you very much
